### DISCUSSION TITLE
Unlimited Previewers Pro Plan

### DISCUSSION THREAD

**Discussion by CDProjects** [vercel/community#4026] [Feature Question] [answered]
### Question

Hello all, just a basic question regarding the Pro Plan - what does it mean by Unlimited Previewers? I intend to build a website for a school and I was wondering if this means someone from the school is able to preview the build while it's ongoing without having to become a team member? Thanks.

**Comment by amyegan** [comment, marked as answer]
Good question, @CDProjects! Having unlimited previewers means you can allow **anyone with the link** to access and comment on Preview Deployment. If you want, you can also secure Preview Deployments by [enabling password protection](https://vercel.com/docs/security/deployment-protection#password-protection). 

By default, only members of your team can **comment** on Preview Deployments. But you can open it to anyone with the link if you update the [Share](https://vercel.com/docs/workflow-collaboration/comments/how-comments-work#sharing) settings.

People you share a Preview Deployment with will ***not*** be automatically added to your Vercel Team

**Comment by CDProjects** [comment]
Thanks for the clarification and info @amyegan, much appreciated 😃
